content/contributing/opencue/Getting started/getting-started-mac.md

```diff
@@ -193,10 +199,10 @@ containing all of the Python components. PyCharm is used here.
    - **Existing environment**
    - **Interpreter** set to `<path to git repository>/venv-dev/bin/python`
 
-1. PyCharm should prompt you to install 'Package requirements'. This will install OpenCue's
+<!-- 1. PyCharm should prompt you to install 'Package requirements'. This will install OpenCue's
```

Comment:
This commented out section can be removed.

Reply:
Sorry about that. That one went under the radar.
Removed in https://github.com/AcademySoftwareFoundation/opencue.io/pull/273/commits/7edbd2922dd8b197d04c0f9cbc18c74d964a485f